types/chain_id_test.go

```diff
@@ -0,0 +1,129 @@
+package types
```

Comment:
maybe could use types_test to avoid import cycle

Reply:
my ide suggests that as well, but there should be no cycle here.

Reply:
fixed